app/services/invoices/payments/stripe_service.rb

```diff
@@ -36,6 +36,7 @@ def update_payment_status(organization_id:, status:, stripe_payment:)
 
         payable_payment_status = payment.payment_provider&.determine_payment_status(payment.status)
         payment.payable_payment_status = payable_payment_status
+        payment.error_code = stripe_payment.error_code if stripe_payment.error_code
```

Comment:
What about something like:
```suggestion
        payment.error_code = stripe_payment.error_code.presence
```

Reply:
I think this would do the same as `payment.error_code = stripe_payment.error_code` 🤔 

I initially wanted to not assign the field if the value is nil to avoid overriding error_code, but maybe it should be overridden if there is no error_code in the webhook content.

Reply:
Hmm, ok. Make sense. Not a blocker for me, feel free to ignore :)